Assets/Scripts/QuestEditor/EditorSelectionList.cs

```diff
@@ -83,7 +84,7 @@ public void SelectItem(UnityEngine.Events.UnityAction call)
         float hOffset = 22;
         foreach (string s in traits)
         {
-            db = new DialogBox(Vector2.zero, new Vector2(10, 1), s);
+            db = new DialogBox(Vector2.zero, new Vector2(10, 1), new StringKey(s,false));
```

Comment:
I think traits can be translated in order to get a full translation App. 
Traits, TileSides, Puzzle Types, Component Types, Token Types,...
all can be used as magic keys inside ini files but translated in screen